### ISSUE TITLE
version 2.4.2 not work on ie 11 because of syntax error

### ISSUE THREAD

**Issue by hkkhchan** [valor-software/ng2-charts#1283]
I found version 2.4.2 not worked on ie11
It is because some syntax not work on ie like
lib/get-color.js line 33 
```
throw new Error(`getColors - Unsupported chart type ${chartType}`);
```
lib/get-color.js line 83 
```
function formatPieColors(colors) {
    return {
        backgroundColor: colors.map((/**
         * @param {?} color
         * @return {?}
         */
        (color) => rgba(color, 0.6))),

```
above case also occur on  ng2-charts.js

I downgrade to older version 1.6.0. now work on ie11.

**Comment by vlapo** [comment]
We had same issue. Downgrade to version 2.3.3 worked for us.

**Comment by DavidDon13** [comment]
I'm having the same issue. Any update on this? Does ng2-charts still support IE11?

**Comment by santam85** [comment]
Released 2.4.3 with a fix for this.
